src/Microsoft.TestPlatform.ObjectModel/Utilities/TypesToLoadAttribute.cs

```diff
@@ -18,6 +17,6 @@ public TypesToLoadAttribute(params Type[] types)
             Types = types;
         }
 
-        public IEnumerable<Type> Types { get; }
+        public Type[] Types { get; }
```

Comment:
This is no longer immutable.

Reply:
IEnumerable, is also not immutable isn't it? But does it matter? How would this collection change to cause problems? We would need a static ctor to find it and modify it on assembly load?